packages/addons/service/transmission/package.mk

```diff
@@ -0,0 +1,60 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="transmission"
+PKG_VERSION="1.0.0"
+PKG_REV="100"
+PKG_ARCH="any"
+PKG_LICENSE="GPLv2"
+PKG_SITE="http://www.libreelec.tv"
+PKG_DEPENDS_TARGET=""
+PKG_AUTORECONF="no"
+PKG_PRIORITY="optional"
+PKG_SECTION=""
```

Comment:
Fine for me, I do not use transmission. Nevertheless, be aware of [this](https://github.com/OpenELEC/unofficial-addons/blob/master/addons/service/downloadmanager/transmission/package.mk#L30).